cli/introspection/src/commands/Init.ts

```diff
@@ -1,11 +1,35 @@
 import { Command, arg, format } from '@prisma/cli'
 import { isError } from 'util'
-import { initPrompt } from '../prompt/initPrompt'
 import fs from 'fs'
 import path from 'path'
 import chalk from 'chalk'
 import { printError, printFix } from '../prompt/utils/print'
 
+export const defaultSchema = `// This is your Prisma schema file,
+// Learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// The \`datasource\` block is used to specify the connection to your DB.
+// Set the \`provider\` field to match your DB type: "postgresql", "mysql" or "sqlite".
+// The \`url\` field must contain the connection string to your DB.
+// Learn more about connection strings for your DB: https://pris.ly/connection-strings
+datasource db {
+provider = "postgresql" // other options are: "mysql" and "sqlite"
```

Comment:
should probably be indented

Reply:
Yes, waiting for final version to fix :)